src/helpers/gho-launch/tests/AaveV3GHOLaunchTest.sol

```diff
@@ -0,0 +1,407 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IPool as IPool_CCIP} from 'src/interfaces/ccip/tokenPool/IPool.sol';
+import {IClient} from 'src/interfaces/ccip/IClient.sol';
+import {IInternal} from 'src/interfaces/ccip/IInternal.sol';
+import {AaveV3GHOLaneTest} from './AaveV3GHOLaneTest.sol';
+import {GhoCCIPChains} from '../constants/GhoCCIPChains.sol';
+import {IACLManager} from 'aave-address-book/AaveV3.sol';
+import {IGhoToken} from 'src/interfaces/IGhoToken.sol';
+import {IOwnable} from 'aave-address-book/common/IOwnable.sol';
+import {IUpgradeableBurnMintTokenPool_1_5_1} from 'src/interfaces/ccip/tokenPool/IUpgradeableBurnMintTokenPool.sol';
+import {IGhoAaveSteward} from 'src/interfaces/IGhoAaveSteward.sol';
+import {IGhoCcipSteward} from 'src/interfaces/IGhoCcipSteward.sol';
+import {IPool} from 'aave-address-book/AaveV3.sol';
+
+abstract contract AaveV3GHOLaunchTest_PreExecution is AaveV3GHOLaneTest {
+  IACLManager public immutable LOCAL_ACL_MANAGER;
+  address public immutable LOCAL_OWNER;
+
+  constructor(
+    GhoCCIPChains.ChainInfo memory localChainInfo,
+    string memory rpcAlias,
+    uint256 blockNumber
+  )
+    AaveV3GHOLaneTest(
+      localChainInfo,
+      GhoCCIPChains.ChainInfo(
+        0,
+        address(0),
+        address(0),
+        address(0),
+        address(0),
+        address(0),
+        address(0),
+        address(0),
+        address(0),
+        address(0)
+      ), // Nullified remote chain info as it's not used for this test
+      rpcAlias,
+      blockNumber
+    )
+  {
+    LOCAL_ACL_MANAGER = IACLManager(localChainInfo.aclManager);
+    LOCAL_OWNER = localChainInfo.owner;
+  }
+
+  /**
+   * @dev executes the generic test suite including e2e and config snapshots
+   */
+  function test_defaultProposalExecution() public virtual {
```

Comment:
this test i think provably makes sense to just have it at the child directly. as for example in case of ink, we will not have a pool, so does not make sense to run it. (and have all these pool dependencies here)

Reply:
I ended up removing it. And it can be added manually if needed like we did in this proposal for the remote lanes.

Change done here: https://github.com/bgd-labs/aave-proposals-v3/pull/802/commits/0894e74332f1042cd1e1a4db7e58e6ebd2cfec49 https://github.com/bgd-labs/aave-proposals-v3/pull/802/commits/58a7f269606ee32167d054eb75bf622ba288d74a